### PR TITLE
Bump versions and add automatic releases

### PR DESCRIPTION
**Pull request by oscarbenjamin**
Work in progress...

### CODE REVIEW

**Comment by edgarcosta** [comment]
It also might be worth it to go all the way to [3.1.3-p1](https://github.com/flintlib/flint/compare/v3.1.2...v3.1.3-p1)

**Comment by oscarbenjamin** [comment]
> It also might be worth it to go all the way to [3.1.3-p1](https://github.com/flintlib/flint/compare/v3.1.2...v3.1.3-p1)

Why is it called 3.1.3-p1?

**Comment by edgarcosta** [comment]
I could have called it 3.1.4, and perhaps should have, but the only thing that was fixed was updating the VERSION before bootstrapping.
https://github.com/Homebrew/homebrew-core/pull/166490#issuecomment-2123447266

**Comment by oscarbenjamin** [comment]
Ah, okay. The website doesn't have anything past 3.1.2:
https://flintlib.org/downloads.html

**Comment by edgarcosta** [comment]
That is indeed true, and someone already created an issue: https://github.com/flintlib/flint/issues/2022

File: .github/workflows/buildwheel.yml [hunk 1]
```diff
@@ -155,3 +155,33 @@ jobs:
...
+  # Deploy wheels and sdist to PyPI
+
+  pypi_release:
+    name: Publish to PyPI
+    needs: [build_wheels, build_sdist]
+    # Run only when a tag is pushed to the flintlib/python-flint repo
+    if: "github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && github.owner == 'flintlib'"
+    environment:
+      name: pypi
+      url: https://pypi.org/p/sympy
```
Comment (oscarbenjamin) [inline .github/workflows/buildwheel.yml:168]: Once this is merged I will test pushing a tag to trigger release.

Comment (oscarbenjamin) [reply .github/workflows/buildwheel.yml:168]: I might need a few attempts while fixing up the Actions workflow.

I guess it might work first time...

**Comment by oscarbenjamin** [comment]
I've just pushed the tag and the workflow is running:
https://github.com/flintlib/python-flint/actions/runs/9750608228

**Comment by oscarbenjamin** [comment]
(It'll take about 30 minutes to reach the point where I can approve deployment.)

**Comment by oscarbenjamin** [comment]
Didn't work. Trying again after gh-153:
https://github.com/flintlib/python-flint/actions/runs/9750919090

**Comment by oscarbenjamin** [comment]
New try: gh-154:
https://github.com/flintlib/python-flint/actions/runs/9751446766

**Comment by oscarbenjamin** [comment]
Seems to be working so after gh-155 I think it will work:
https://github.com/flintlib/python-flint/actions/runs/9751640936

File: README.md [hunk 2]
```diff
@@ -133,8 +133,19 @@ CHANGELOG
 
 Next release:
 
+- [gh-148](https://github.com/flintlib/python-flint/pull/148)
+  Remove debug symbols to make smaller Linux binaries.
+- [gh-144](https://github.com/flintlib/python-flint/pull/144)
+  Add `rel_one_ccuracy_bits` to `arb` and `acb`.
+- [gh-142](https://github.com/flintlib/python-flint/pull/142)
+  Add `acb_theta` (only available for Flint >= 3.1).
```
Comment (oscarbenjamin) [inline README.md:141]: I don't know much about what `acb_theta` is but I get the impression that it warrants a more significant release note (that someone else would need to write).

**Comment by oscarbenjamin** [comment]
Okay here you go Edgar:
https://pypi.org/project/python-flint/0.7.0a2/#files

`pip install --pre python-flint` will give all the `acb_theta` you want!

File: .github/workflows/buildwheel.yml (hunk 1 above)
Comment (oscarbenjamin) [inline .github/workflows/buildwheel.yml:168]: Looks I forgot to change the URL when adapting this code...

**Pull request merged**